spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaPropertiesBaseWithBootstrapServersAndProducerConsumer.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.kafka;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+/**
+ * Common Spring Kafka configuration properties - including bootstrap servers and producer and consumer
+ * Kafka client properties.
+ *
+ * @author Chris Bono
+ * @since 2.7.0
+ */
+class KafkaPropertiesBaseWithBootstrapServersAndProducerConsumer extends KafkaPropertiesBaseWithBootstrapServers {
```

Comment:
ℹ️ This hierarchy is a bit forced and brittle by nature of the config properties method / naming convention. Using inheritance makes the get/set available. Otherwise I would have explored composition.